src/components/PostPageDetail/elements/Content.js

```diff
@@ -84,12 +85,26 @@ const Content = ({
     }
   }, [containerHeight, textHeight]);
 
+  const randomIndex = React.useMemo(() => {
+    return Math.floor(Math.random() * listFeedColor.length);
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation of `randomIndex` using `Math.random()` is correct. However, keep in mind that this will generate a new random index every time the component re-renders. If you want the same random index for the lifetime of the component, this approach is fine. Otherwise, you might need to adjust your logic.
```diff
+ const randomIndex = React.useMemo(() => {
+   return Math.floor(Math.random() * listFeedColor.length);
+ }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->